webapp/packages/core-connections/src/Administration/Connections/ConnectionForm/ConnectionFormController.ts

```diff
@@ -99,14 +105,61 @@ implements IInitializableController {
     }
   };
 
+  private isConnectionNameAlreadyExists(name: string) {
+    for (const connection of this.connectionsResource.data.values()) {
+      if (connection.id !== this.model.connection.id && connection.name === name) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  private valdiate() {
+    const status = { passed: true, errorMsg: '' };
```

Comment:
add type, also seems it's not `passed` but `status`, `errorMsg` (please don't use abbreviations) is `message`